_posts/2020-01-01-2019-summary.md

```diff
@@ -0,0 +1,230 @@
+  optimizations in recent times and we will be blogging about this more in coming weeks.
+* [Unwrap cast/predicate pushdown]({% post_url 2019-05-21-optimizing-the-casts-away %}) optimizations.
+* Add support for pushing down operations such as limit, table sample, or projections into connectors during
+  query planning. This allows connectors to optimize how data is accessed before it's provided to the 
+  Presto engine for further processing. 
+* [Dynamic filtering]({% post_url 2019-06-30-dynamic-filtering %}).
+* Cost-Based Optimizer can now consider [estimated query peak memory](https://github.com/prestosql/presto/pull/247) 
+  footprint. This is especially useful for optimizing bigger queries, where not all parts of the query can 
+  be run concurrently.
+* Improved handling of [projections](https://github.com/prestosql/presto/pull/1431), 
+  [aggregations](https://github.com/prestosql/presto/pull/864) and [cross joins](https://github.com/prestosql/presto/pull/1359) 
+  in cost based optimizer.
+* Improved accounting and reporting of physical and network data read or transmitted during query processing.    
+
+## Performance
+
+* [10x performance improvement for `UNNEST`]({% post_url 2019-08-23-unnest-operator-performance-enhancements %}).
+* 2-7x improvement in performance of [ORC decoders]({% post_url 2019-04-23-even-faster-orc %}), resulting in a 
+  10% global CPU improvement for the TPC-DS benchmark.
+* Improvements when reading small Parquet files, files with large number of columns, or files with small row
+  groups.
+* Support for new ORC bloom filters.
+* Remove [redundant `ORDER BY`]({% post_url 2019-06-03-redundant-order-by %}) clauses.
+* Improvements for [`IN` / `NOT-IN`]({% post_url 2019-06-03-redundant-order-by %}) with subquery expressions (i.e., semijoin).
+* Huge performance improvements when [reading from `information_schema`](https://github.com/prestosql/presto/pull/1329).
+* Reduce query latency and Hive Metastore load.
+* Improve metadata handling during planning. This can result in dramatic improvements in latency, 
+  especially for connectors such as MySQL, PostgreSQL, Redshift, SQL Server, etc. Some queries like 
+  `SHOW SCHEMAS`/`SHOW TABLES` that could take several minutes to complete now finish in a few seconds.
+* Improved stability and performance when spilling is enabled.  
+
+## Functions
+
+* [`combinations`](/docs/current/functions/array.html#combinations)
+* [`format`](/docs/current/functions/conversion.html#format)
+* [`UUID` type](/docs/current/functions/uuid.html) and related functions.
+* [`all_match`](/docs/current/functions/array.html#all_match),
+  [`any_match`](/docs/current/functions/array.html#any_match) and 
+  [`none_match`](/docs/current/functions/array.html#none_match).
+* Support for higher-order [`reduce_agg`](/docs/current/functions/aggregate.html#reduce_agg) 
+  function in aggregation queries.
```

Comment:
maybe instead of "higher-order" use wording easier to digest:

```
* Support flexible aggregation with lambdas using
  [`reduce_agg`](/docs/current/functions/aggregate.html#reduce_agg).
```